cmd/mesh/manifest.go

```diff
@@ -146,7 +146,7 @@ func genApplyManifests(setOverlay []string, inFilename string, force bool, dryRu
 			l.logAndPrintf("\n%s\n%s", cs, strings.Repeat("=", len(cs)))
 		}
 
-		if strings.TrimSpace(out[cn].Stderr) != "" {
+		if !isStderrOkay(out[cn].Stderr) {
```

Comment:
Consider renaming function to ignoreError, which is more meaningful.

Reply:
Fixed.